src/django_htmx/management/commands/downloadhtmx.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import TYPE_CHECKING
+from typing import Any
+from urllib.parse import urlparse
+from urllib.parse import urlunparse
+from urllib.request import urlopen
+
+from django.conf import settings
+from django.core.checks import Tags
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+if TYPE_CHECKING:
+    from argparse import ArgumentParser
+
+
+class Command(BaseCommand):
+    help = "Download the given htmx version and the extensions."
+    requires_system_checks = [Tags.staticfiles]
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        parser.add_argument(
+            "version",
+            nargs="?",
+            help="version to download, e.g. '1.0.1. default: latest",
+        )
+        parser.add_argument(
+            "--dest",
+            "-d",
+            type=Path,
+            default=None,
+            help="where to download HTMX to. default: the first STATICFILES_DIR",
```

Comment:
I’d rather we didn’t have a default, every project has a different static layout.

Reply:
Should the script just accept a target directory, then?